front/app/containers/IdeasNewSurveyPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -188,10 +190,23 @@ const IdeasNewSurveyForm = ({ project, phaseId }: Props) => {
       publication_status: data.publication_status || 'published',
     };
 
+    const handleOnError = () => {
```

Comment:
I also wanted to make it more generic and work in all possible cases now and in the future. E.g., a user opens this page, then posting gets disabled (BE returns the `posting_disabled` reason), and then the user submits the survey. With this code, it will be nicely handled because we do not check any specific reason. Though I agree it hides a bit the original intention of this callback, but it covers more use cases.

Reply:
But I'm not 100% sure about it, so please let me know if you don't agree :slightly_smiling_face:

Reply:
Hm maybe. But on the other hand, currently, it nicely handles all possible errors and lets another screen handle them. So, all errors should be handled and mentioned only on the other screen, making this `handleOnError` function more resilient to changes (no changes needed when adding new errors).